presto-product-tests/conf/docker/common/compose-commons.sh

```diff
@@ -23,7 +23,7 @@ function export_canonical_path() {
 
 source ${BASH_SOURCE%/*}/../../../bin/locations.sh
 
-export DOCKER_IMAGES_VERSION=${DOCKER_IMAGES_VERSION:-12}
+export DOCKER_IMAGES_VERSION=${DOCKER_IMAGES_VERSION:-13}
```

Comment:
Please open a separate PR for this migration. And make it be merges as soon as possible.